ansible/roles/proteus-events/templates/proteus-events.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Proteus Events service
+
+[Service]
+User={{ proteus_user }}
+Group={{ proteus_group }}
+ExecStart={{ proteus_path }}/proteus-events --config {{ proteus_config_path }}/proteus-events.toml start
```

Comment:
I mean, is there any reason for these services to share unix user? As far as I see, there is no -- these services share no objects.

I'm sorry for the form of the question :-)